host_vars/pulsar-high-mem1/pulsar-high-mem1.yml

```diff
@@ -0,0 +1,73 @@
+hostname: "{{ ansible_hostname }}.usegalaxy.org.au"
+
+# Keys and shares
+
+create_ssh_key: yes  # Only the first time.
+ssl_country: "AU"
+ssl_state: "Vic"
+ssl_location: "Melbourne"
+ssl_organisation: "Galaxy Australia"
+ssl_operational_unit: "Pulsar High Mem1"
+ssl_email: "help@genome.edu.au"
+
+#host specific pulsar settings
+
+rabbitmq_password_galaxy_au: "{{ vault_rabbitmq_password_galaxy_hm1_pawsey }}"
+pulsar_queue_url: "pawsey-queue.usegalaxy.org.au"
+pulsar_rabbit_username: "galaxy_hm1"
+pulsar_rabbit_vhost: "/pulsar/galaxy_hm1"
+
+attached_volume_device: /dev/vdb
```

Comment:
attached_volumes:
  - device: /dev/vdb
    path: /mnt
    fstype: ext4

Reply:
Actually, I have to remove that section altogether as I had to construct an LVM for it instead. :( `/dev/mapper/vg1-lvol0 on /mnt type ext4 (rw,relatime)`